src/java/org/apache/cassandra/utils/DiagnosticSnapshotService.java

```diff
@@ -139,7 +140,7 @@ private void maybeTriggerSnapshot(TableMetadata metadata, String prefix, Iterabl
         long now = nanoTime();
         AtomicLong cached = lastSnapshotTimes.computeIfAbsent(metadata.id, u -> new AtomicLong(0));
         long last = cached.get();
-        long interval = Long.getLong("cassandra.diagnostic_snapshot_interval_nanos", SNAPSHOT_INTERVAL_NANOS);
+        long interval = DIAGNOSTIC_SNAPSHOT_INTERVAL_NANOS.getLong(SNAPSHOT_INTERVAL_NANOS);
```

Comment:
`SNAPSHOT_INTERVAL_NANOS` seems to be OK to move to CassandraRelevantProperties as default value as it is done for similar properties there, since it is private in this class and not used anywhere else but here.

Reply:
Fixed.